package.json

```diff
@@ -69,62 +95,71 @@
     "@types/react-dom": "16.9.17",
     "@types/semver": "7.3.13",
     "@types/sinon": "9.0.11",
-    "@types/terser-webpack-plugin": "5.0.4",
     "@types/tmp": "0.2.3",
-    "@types/webpack-node-externals": "2.5.3",
-    "analytics-client": "^2.0.1",
-    "axios": "^0.27.2",
+    "@vercel/webpack-asset-relocator-loader": "^1.7.3",
     "chai": "4.3.7",
-    "copy-webpack-plugin": "7.0.0",
     "css-loader": "5.2.7",
-    "d3": "4.13.0",
-    "debug": "4.3.4",
     "electron": "^25.8.2",
-    "electron-builder": "^23.6.0",
+    "electron-forge-resource-plugin": "^1.0.0-alpha.6",
     "electron-mocha": "^11.0.2",
     "electron-notarize": "1.2.2",
     "electron-rebuild": "^3.2.9",
-    "electron-updater": "5.3.0",
-    "esbuild-loader": "2.20.0",
-    "etcher-sdk": "8.3.1",
     "file-loader": "6.2.0",
     "husky": "4.3.8",
-    "i18next": "21.10.0",
-    "immutable": "3.8.2",
     "lint-staged": "10.5.4",
-    "lodash": "4.17.21",
     "mini-css-extract-plugin": "1.6.2",
     "mocha": "^9.1.1",
     "native-addon-loader": "2.0.1",
-    "node-ipc": "9.2.1",
+    "node-loader": "^2.0.0",
     "omit-deep-lodash": "1.1.7",
-    "outdent": "0.8.0",
-    "path-is-inside": "1.0.2",
     "pkg": "^5.8.1",
-    "pnp-webpack-plugin": "1.7.0",
-    "pretty-bytes": "5.6.0",
-    "react": "16.8.5",
-    "react-dom": "16.8.5",
-    "react-i18next": "11.18.6",
-    "redux": "4.2.0",
-    "rendition": "19.3.2",
-    "semver": "7.3.8",
-    "simple-progress-webpack-plugin": "1.1.2",
     "sinon": "9.2.4",
     "string-replace-loader": "3.1.0",
     "style-loader": "2.0.0",
-    "styled-components": "5.3.6",
-    "sys-class-rgb-led": "3.0.1",
-    "terser-webpack-plugin": "5.3.6",
-    "ts-loader": "8.4.0",
-    "ts-node": "9.1.1",
+    "ts-loader": "^9.5.0",
+    "ts-node": "^10.9.1",
     "tslib": "2.4.1",
     "typescript": "4.4.4",
-    "url-loader": "4.1.1",
-    "uuid": "8.3.2",
-    "webpack": "5.75.0",
-    "webpack-cli": "4.10.0",
-    "webpack-dev-server": "4.11.1"
+    "url-loader": "4.1.1"
+  },
+  "hostDependencies": {
```

Comment:
Not sure how good this choice will turn out to be, but I wanted to avoid what we declare as app dependencies for deb to get out of sync with what we install on CI, so this will have to do for now.